src/commands/test/lib/run-tests-command.ts

```diff
@@ -176,6 +180,38 @@ export abstract class RunTestsCommand extends AppCommand {
     }
   }
 
+  private sortDeviceSets(a: DeviceSet, b: DeviceSet): number {
+    if (a.name > b.name) {
+      return 1;
+    }
+    if (a.name < b.name) {
+      return -1;
+    }
+    return 0;
+  }
+
+  private async promptDevices(client: AppCenterClient): Promise<void> {
+    let configs: DeviceSet[] = await client.test.listDeviceSetsOfOwner(this.app.ownerName, this.app.appName);
+    // Sort devices list like it was done on AppCenter Portal
+    configs = configs.sort(this.sortDeviceSets);
+    const choices = configs.map((config: DeviceSet) => {
+      return {
+        name: config.name,
+        value: config.slug
+      };
+    });
+    const questions: Questions = [
+      {
+        type: "list",
+        name: "deviceSlug",
+        message: "Pick a device set to use",
+        choices: choices
+      }
+    ];
+    const answers: any = await prompt.question(questions);
```

Comment:
Is there a chance we can try to use something more specific than `any`?

Reply:
I don't think so